tests/models/realm/test_modeling_realm.py

```diff
@@ -480,15 +480,12 @@ def test_inference_encoder(self):
     def test_inference_open_qa(self):
         from transformers.models.realm.retrieval_realm import RealmRetriever
 
-        config = RealmConfig()
```

Comment:
Not sure why it is done this way before, but using `from_pretrained` without manually `config` seems more cleaner - and fixes the CI.